code/style.css

```diff
@@ -1,4 +1,213 @@
-html {
-  background: #0025ff;
-  color: #fff;
+body {
```

Comment:
You have a good structure in your css file. It´s easy to find what you are looking for!